### DISCUSSION TITLE
rename une guilde

### DISCUSSION THREAD

**Discussion by kekeishere** [Crownicles/Crownicles#1787] [Suggestions fermées]
Je propose que pour les chefs de guilde il y a une option pour changer le nom en échange d'une certaine somme histoire que l on ai pas a recréer une autre guilde pour changer de nom

**Comment by Nargacaura** [comment]
Uhmm... Pourquoi est-ce mis dans "accepté"? Et je suis sûr qu'il y a déjà une discussion dessus... (même 2: les #869 et #580.)

**Comment by BastLast** [comment]
Je ferme du coup. Go voter pr la plus ancienne si l'idée vous intéresse
